Decode and encode (#572)
linear-b/gitstream: docs/filter-functions.md

| [`allDocs`](#alldocs)<br />Checks the list includes only documents                                                                                          | [`files`](./context-variables.md#files)                    | -                            | Bool                 |
| [`allImages`](#allimages)<br />Checks the list includes only images                                                                                         | [`files`](./context-variables.md#files)                    | -                            | Bool                 |
| [`allTests`](#alltests)<br />Checks the list includes only tests                                                                                            | [`files`](./context-variables.md#files)                    | -                            | Bool                 |
| [`codeExperts`](#codeexperts)<br />Get list of contributors based on expert reviewer model results                                                          | [`repo`](./context-variables.md#repo)                      | `gt`, `lt`                   | [String]             |

> | [`decode`](#decode)<br />Decode data from b64                                                                                                               | String (b64 encoded)                                       | -                            | Object               |

Reviewer: ```suggestion
| [`decode`](#decode)<br />Decode data from Base64                                                                                                               | String (Base64 encoded)                                       | -                            | Object               |
```

> | [`encode`](#encode)<br />Encode data into b64 representation                                                                                                | Object                                                     | -                            | String (b64 encoded) |

Reviewer: ```suggestion
| [`encode`](#encode)<br />Encode data into Base64 representation                                                                                                | Object                                                     | -                            | String (Base64 encoded) |
```

          reviewers: {{ repo | codeExperts(gt=10) }}
```

#### `decode`

> Decode base64 string into and object

Reviewer: ```suggestion
Decode Base64 string into and object
```

<div class="filter-details" markdown=1>

| Argument | Usage  | Type   | Description                         |
| -------- | ------ | ------ | ----------------------------------- |
| -        | Input  | String | Base64 representation of the object |
| -        | Output | Object | Decoded objet                       |

</div>

```yaml+jinja
{{ base64String | decode }}
```

#### `encode`

> Encode data into base64 representation. When an encoded string is passed as input for [`add-comment`](./automation-actions.md#add-comment), the action automatically detects and decodes it.

Reviewer: ```suggestion
Encode data into Base64 representation. When an encoded string is passed as input for [`add-comment`](./automation-actions.md#add-comment), the action automatically detects and decodes it.
```

<div class="filter-details" markdown=1>

| Argument | Usage  | Type         | Description                         |
| -------- | ------ | ------------ | ----------------------------------- |
| -        | Input  | Object       | The input object to encode          |

> | -        | Output | String (b64) | Base 64representation of the object |

Reviewer: ```suggestion
| -        | Output | String (Base64) | Base64 encoding of the object |
```

</div>

```yaml+jinja

> {{ "String" | encode }}

Reviewer: I would add a meaningful usage as example as this one doesnt say nothing on how to use it, it could be any filter...

Author: Added better example
